### ISSUE TITLE
 Problem reading the .txt files of pumping curves and storages in shp_to_inp function

### ISSUE THREAD

**Issue by ghost** [dleutnant/swmmr#41]
Hello all, I need to insert a series of pumping and storage curves using the ```shp_to_inp``` function: I created a .txt file containing all the pump curves according to the structure of the manual but only the first curve is read, can anyone help me? I attach the .txt file in question, my .inp file and my code, thank you.

[Paullo inp.txt](https://github.com/dleutnant/swmmr/files/4449925/Paullo.inp.txt)
[Paullo_Pcurve.txt](https://github.com/dleutnant/swmmr/files/4449866/Paullo_Pcurve.txt)
```
if (!require(swmmr)) devtools::install_github("dleutnant/swmmr")
library(tidyverse)
library(swmmr)
library(purrr)
library(dplyr)
library(sf)

out_dir <- "C:/Users/Enrico/Tesi/Paullo"

PAULLO <- shp_to_inp(
  path_options = file.path(out_dir,"SWMM/Paullo_Opzioni.txt"),
  path_timeseries = file.path(out_dir,"PIOGGE/Paullo_TS1.dat"),
  path_polygon = file.path(out_dir,"SWMM/Paullo_polygon.shp"),
  path_line = file.path(out_dir,"SWMM/Paullo_link.shp"),
  path_point = file.path(out_dir,"SWMM/Paullo_point.shp"),
  path_outfall = file.path(out_dir,"SWMM/Paullo_outfall.shp"),
  path_pumps = file.path(out_dir,"SWMM/Paullo_pumps.shp"),
  path_pump_curve = file.path(out_dir,"SWMM/Paullo_Pcurve.txt"),
  path_storage = file.path(out_dir,"SWMM/Paullo_storages.shp"),
  path_storage_curve = file.path(out_dir,"SWMM/Paullo_Scurve.txt"),
)

summary(PAULLO)

write_inp(PAULLO, file.path(out_dir, "PAULLO.inp"))
```

**Comment by Enrilok** [comment]
Hello all, I'm sorry to disturb you again but unfortunately I still haven't been able to solve my problem. I try to explain myself better: I need to insert five different pumping curves and numerous storage curves through the ```shp_to_inp``` function, so I created a .txt file without header with the following structure:

```"Curve name" "Pump2 (in my case)" "x" "y"```

and I inserted all five pumping curves within the same .txt file, but only the first one is read. Do I need to create a single .txt file for each pumping and storage curve? Can anyone tell me where I'm wrong? I attach my code, the .inp file and the .txt file containing the pumping curves. Thanks a lot,
Enrico

[Paullo_Pcurve.txt](https://github.com/dleutnant/swmmr/files/4479833/Paullo_Pcurve.txt)
[Paullo.inp.txt](https://github.com/dleutnant/swmmr/files/4479835/Paullo.inp.txt)
```if (!require(swmmr)) devtools::install_github("dleutnant/swmmr")
library(tidyverse)
library(swmmr)
library(purrr)
library(dplyr)
library(sf)

out_dir <- "C:/Users/Enrico/Tesi/Paullo"

PAULLO <- shp_to_inp(
  path_options = file.path(out_dir,"SWMM/Paullo_Opzioni.txt"),
  path_timeseries = file.path(out_dir,"PIOGGE/Paullo_TS1.dat"),
  path_polygon = file.path(out_dir,"SWMM/Paullo_polygon.shp"),
  path_line = file.path(out_dir,"SWMM/Paullo_link.shp"),
  path_point = file.path(out_dir,"SWMM/Paullo_point.shp"),
  path_outfall = file.path(out_dir,"SWMM/Paullo_outfall.shp"),
  path_pumps = file.path(out_dir,"SWMM/Paullo_pumps.shp"),
  path_pump_curve = file.path(out_dir,"SWMM/Paullo_Pcurve.txt"),
  path_storage = file.path(out_dir,"SWMM/Paullo_storages.shp"),
  path_storage_curve = file.path(out_dir,"SWMM/Paullo_Scurve.txt"),
)
```

**Comment by Enrilok** [comment]
Hello, sorry to disturb you again but I found the problem related to reading the .txt file containing the pump curves using the ```shp_to_inp``` function: in my case I have to insert 5 different pump curves and if I create the .txt file according to the following structure from you indicated in the swmmr manual:
![11](https://user-images.githubusercontent.com/60235002/80617074-f23fdb80-8a41-11ea-987f-fa23bdf86b86.PNG)
The .inp file that is created has the following structure, in which, however, only the first curve is read by SWMM (in my case the P942CURVA_A):
![1](https://user-images.githubusercontent.com/60235002/80617163-0e437d00-8a42-11ea-866f-37b873612c09.PNG)
However, trying to manually edit the .inp file created in this way, the 5 curves are all read by SWMM:
![2](https://user-images.githubusercontent.com/60235002/80617425-5f537100-8a42-11ea-8143-f2704df2b569.PNG)
therefore I think it's necessary to modify the structure of the .txt file which will have to be read by the ```shp_to_inp``` function indicating the type of pump (in my case Pump2) in correspondence with the name and the first data of each curve to be inserted, as follows:
![correct](https://user-images.githubusercontent.com/60235002/80618962-582d6280-8a44-11ea-8316-1d315cc918bb.PNG)
I attach the original .inp file and the one edited manually, hoping that it will help to solve this problem. Thanks a lot and I apologize again for the inconvenience,
Enrico
[Paullo_ORIGINAL.txt](https://github.com/dleutnant/swmmr/files/4552913/Paullo_ORIGINAL.txt)
[Paullo_CORRECT.txt](https://github.com/dleutnant/swmmr/files/4552914/Paullo_CORRECT.txt)

**Comment by hsonne** [comment]
Thanks, "ghost" for giving this valuable error report. I hopefully fixed the issue with #100. The changes are now available on the `dev` branch.
